src/Microsoft.TemplateEngine.Cli/TemplateInvoker.cs

```diff
@@ -170,6 +164,23 @@ private async Task<CreationResultStatus> CreateTemplateAsync(ITemplateMatchInfo
                 return CreationResultStatus.CreateFailed;
             }
 
+            string fallbackName = new DirectoryInfo(_commandInput.OutputPath ?? Directory.GetCurrentDirectory()).Name;
```

Comment:
should the current directory be used not only if `_commandInput.OutputPath` is null but also length 0?

Reply:
this logic existed before, however I'm not sure if commandline library handles these values to be non-empty. 
added extra check to be on safe side.